CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.58.0      2024-08-27
+
+* Log Insights:
```

Comment:
Might be nice to say something as a title, like "Log Insights: Update log parsing mechanism" or "Log Insights: Introduce a new log parsing mechanism". Uhmm both don't sound so good, but it'd be nice to have a title.